src/sh/install/wsl.sh

```diff
@@ -35,6 +35,11 @@ fi
 # make sure the font dir exists
 mkdir -p "${font_dir}" 1>/dev/null
 
+# ensure unzip exists
+if ! command -v unzip >/dev/null 2>&1; then
+	sudo apt install -y unzip
```

Comment:
this should happen in the impacted linux distro (which is evaluated first); in this case, within debian.